<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>611</width>
    <height>537</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="grafica">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>414</width>
      <height>393</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tap">
     <attribute name="title">
      <string>Agregar</string>
     </attribute>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>PARTICULAS</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>ID</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="ID_spinBox"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>ORIGEN X</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="ORIGEN_X_spinBox"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>ORIGEN Y</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSpinBox" name="ORIGEN_Y_spinBox"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="x_label">
           <property name="text">
            <string>DESTINO X</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QSpinBox" name="x_spinBox">
           <property name="maximum">
            <number>500</number>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="y_label">
           <property name="text">
            <string>DESTINO Y</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QSpinBox" name="y_spinBox">
           <property name="maximum">
            <number>500</number>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="velocidad_label">
           <property name="text">
            <string>VELOCIDAD</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QSpinBox" name="velocidad_spinBox">
           <property name="maximum">
            <number>1000</number>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="rojo_label">
           <property name="text">
            <string>ROJO</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QSpinBox" name="rojo_spinBox">
           <property name="maximum">
            <number>255</number>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="verde_label">
           <property name="text">
            <string>VERDE</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QSpinBox" name="verde_spinBox"/>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="azul_label">
           <property name="text">
            <string>AZUL</string>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QSpinBox" name="azul_spinBox"/>
         </item>
         <item row="9" column="0" colspan="2">
          <widget class="QPushButton" name="FINAL_pushButton">
           <property name="text">
            <string>AGREGAR AL FINAL</string>
           </property>
          </widget>
         </item>
         <item row="10" column="0" colspan="2">
          <widget class="QPushButton" name="inicio_pushButton">
           <property name="text">
            <string>AGREGAR AL INICIO</string>
           </property>
          </widget>
         </item>
         <item row="11" column="0" colspan="2">
          <widget class="QPushButton" name="mostrar_pushButton">
           <property name="text">
            <string>MOSTRAR</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPlainTextEdit" name="salida"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tabla</string>
     </attribute>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0" colspan="3">
       <widget class="QTableWidget" name="tableWidget"/>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="orden_dist">
        <property name="text">
         <string>Ordenar por distancia</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="Buscar_lineEdit">
        <property name="placeholderText">
         <string>ID</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="mostrar_tabla_pushButton">
        <property name="text">
         <string>Mostrar</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="buscar_pushButton">
        <property name="text">
         <string>Buscar</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="orden_vel">
        <property name="text">
         <string>Ordenar por velocidad</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QPushButton" name="orden_id">
        <property name="text">
         <string>Ordenar por ID</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Grafica</string>
     </attribute>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0" colspan="2">
       <widget class="QGraphicsView" name="graphicsView"/>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="Dibujar">
        <property name="text">
         <string>Dibujar</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="limpiar">
        <property name="text">
         <string>Limpiar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>611</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionAbrir"/>
    <addaction name="actionGuardar"/>
   </widget>
   <addaction name="menuArchivo"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbrir">
   <property name="text">
    <string>Abrir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionGuardar">
   <property name="text">
    <string>Guardar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
